src/binder/expression/agg_call.rs

```diff
@@ -56,6 +56,12 @@ impl std::fmt::Debug for BoundAggCall {
     }
 }
 
+impl std::fmt::Display for BoundAggCall {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}({}) -> {}", self.kind, self.args[0], self.return_type)
```

Comment:
Agg call could have zero or >1 args, and therefore we should format and join all args together.

Reply:
e.g., count(*) have 0 args, and string_agg (not implemented yet) has 2 args.